packages/@react-spectrum/list/src/styles.css

```diff
@@ -56,13 +56,6 @@
         pointer-events: none;
       }
 
-      /* Box shadow for bottom border for non-selected rows that is immmediately above a selected row. */
-      &.is-next-selected {
-        &:after {
-          box-shadow: inset 0 -1px 0 0 var(--spectrum-global-color-blue-500);
-        }
-      }
```

Comment:
Minor note, potentially ignorable:
By simplifying this, a wiggle case has returned:
1. Go to https://reactspectrum.blob.core.windows.net/reactspectrum/59530d349cab3e68b3c57100957b6659c7effe87/storybook/index.html?path=/story/listview--selection-multiple-checkbox and select the second row.
2. Repeatedly select and deselect the first row. Note that the border between the two rows wiggles

I'm fine with ignoring it in favor of simpler css, I probably overengineered this css to avoid this wiggle in the first place haha

Reply:
Ok, I'll have a look at this separately. good find though